mongodb-graphite.gemspec

```diff
@@ -3,19 +3,20 @@ $:.push File.expand_path("../lib", __FILE__)
 
 Gem::Specification.new do |s|
   s.name        = "mongodb-graphite"
-  s.version     = '1.1.1'
+  s.version     = '1.1.2'
   s.platform    = Gem::Platform::RUBY
   s.authors     = ["Grégoire Seux"]
   s.email       = ["g.seux@criteo.com"]
   s.homepage    = "https://github.com/kamaradclimber/mongodb-graphite"
-  s.summary     = %q{}
-  s.description = %q{}
-  s.files         = `git ls-files`.split("\n")
-  s.test_files    = `git ls-files -- test/{functional,unit}/*`.split("\n")
-  s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
+  s.summary     = %q(Simple gem to provide graphite with mongodb server stats)
+  s.description = %q(This gem retrieves detailed server stats from mongo instances, and pushes the information to graphite)
+  s.files       = `git ls-files`.split("\n")
+  s.test_files  = `git ls-files -- test/{functional,unit}/*`.split("\n")
+  s.executables = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
```

Comment:
All the `git ls-files` here can be expensive when bundling a gem from source as it is shelling out three times.

You also need to be careful using `bin` as the executable dir since binstubs often go in there. The ['new' convention is `exe`](http://bundler.io/blog/2015/03/20/moving-bins-to-exe.html) to avoid that possibility. Or you can just specify `bin/mongodb-graphite`

As an aside, rubygems don't really use `test_files` anymore, so it could be left out entirely.

e.g. see https://github.com/bundler/bundler/blob/5e2e2ff089af4875c54b27cfb5d4286471dff125/lib/bundler/templates/newgem/newgem.gemspec.tt#L27-L30

```ruby
  spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
  spec.bindir        = "exe"
  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
  spec.require_paths = ["lib"]
```

notice how `spec.executables` re-uses the result from `spec.files`